### ISSUE TITLE
[FE] Address Validation: Edit address is saving obviously fake addresses

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#4207]
## What happened?

1. From the Edit Mailing Address from, I changed address from "811 Vermont Ave NW, Washington, DC 20571" (real address) to "1234 Is this working street, Washington, DC 20592" (fake address).
2. I was able to successfully save the obviously fake address.

<img width="307" alt="Screen Shot 2019-12-12 at 2 45 55 PM" src="https://user-images.githubusercontent.com/34068740/70744939-cc16da80-1cf0-11ea-9a07-b2e5e73523c2.png">

## Specs:
- Device: Macbook
- Browser: Chrome
- Test User: This happened with user 226.

## Steps to Reproduce
- URL: https://staging.va.gov/profile

See top section for steps to reproduce.

## Desired behavior

Obviously fake addresses should trigger the address validation flow.

## Acceptance Criteria
- [ ] Obviously fake addresses should trigger the address validation flow.

**Comment by Samara-Strauss** [comment]
cc/ @erikphansen @artistgreen @Justin-Pickett for awareness.

**Comment by erikphansen** [comment]
@Samara-Strauss I just took a look at this one again and the behavior has changed slightly due to changes I merged yesterday.

1. Entering `1234 Is this working street, Washington, DC 20592` triggers the validation modal, asking you to confirm that you entered the correct street. It then allows you to select the address as entered and save it. However it should _not_ allow you to select it (the update will fail) but that's due to this bug with VA Profile: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4278
2. If you enter `1234 Is this working street, Washington, DC 20571` (note the zip code), that will save. This seems to have to do with the fact that 20571 is a flagged zip code and validation is not done on that address. I'm not sure the reason, something something government building. More info in this Slack thread https://dsva.slack.com/archives/C7TE0PFTL/p1576264799001000

So I'm closing this now as we already have a ticket for item 1 and item 2 is apparently correct behavior for that zip code.
